jdbc/src/main/java/org/apache/metamodel/jdbc/JdbcMetadataLoader.java

```diff
@@ -383,6 +343,52 @@ private void loadColumns(JdbcTable table, DatabaseMetaData metaData) {
         }
     }
 
+    private void processColumn(JdbcTable table, boolean convertLobs, ResultSet rs, int columnNumber) throws SQLException {
+        final String columnName = rs.getString(4);
+        if (_identifierQuoteString == null && new StringTokenizer(columnName).countTokens() > 1) {
+            logger.warn("column name contains whitespace: \"" + columnName + "\".");
+        }
+
+        final int jdbcType = rs.getInt(5);
+        final String nativeType = rs.getString(6);
+        final Integer columnSize = rs.getInt(7);
+        Integer decimalDigits = rs.getInt(9);
```

Comment:
We should add `final` to this one.

Reply:
Fixed!

Reply:
Doesn't look like it's fixed. Did you push the change?

Reply:
Doh. Pilot error. Look again!